components/server-side-render/index.js

```diff
@@ -80,6 +80,7 @@ export class ServerSideRender extends Component {
 				<Placeholder><Spinner /></Placeholder>
 			);
 		} else if ( response.error ) {
+			// translators: %s: error message describing the problem
 			return (
 				<Placeholder>{ sprintf( __( 'Error loading block: %s' ), response.errorMsg ) }</Placeholder>
```

Comment:
Not sure if in JS it's the same, but ideally the translators comment should be immediately before the line where it is used e..g.:
```
<Placeholder>{ sprintf( 
    // translators: %s: error message describing the problem
    __( 'Error loading block: %s' )
```

@ocean90 ?

Reply:
Hi @afercia thank you for referring this, the code was updated now translator comments are immediately before the line where they are used.